tokio-util/Cargo.toml

```diff
@@ -6,7 +6,7 @@ name = "tokio-util"
 # - Create "tokio-util-0.7.x" git tag.
 version = "0.7.10"
 edition = "2021"
-rust-version = "1.56"
+rust-version = "1.63"
```

Comment:
Just to clarify, is this no accurately representing the minimal version this would compile on? As the issue I raised mentioned it was `1.60` with an older `tokio` release.

I understand with newer `tokio` releases that `1.56` isn't supported, and the current `1.56` for `tokio-util` is already inaccurate as a fallback for toolchains below `1.60`.

---

So is `rust-version` within `tokio` crates likely not to represent a crates actual minimal toolchain version to compile on? But the MSRV for `tokio` crates as a whole?

**EDIT:** My bad, I forgot that a newer release of `tokio-util` also [bumped to `hashbrown 0.14`](https://github.com/tokio-rs/tokio/pull/6102) which has `rust-version = "1.63"` 🤦‍♂️